jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -146,6 +146,8 @@ public void buildDocker()
               projectProperties.getJavaLayerConfigurations().getLayerConfigurations(),
               helpfulSuggestions);
 
+      projectProperties.waitForLoggingThread();
```

Comment:
It seems like we might be doing too much in the class. It's not longer supplier of `properties`. It does seem sort of weird that we would call `waitForLoggingThread()` on it.

This is an interface that appears it is feeding some data back to jib-core under normal circumstances? But `waitForLoggingThread` is an action that appears to have nothing to do with jib-core?

Reply:
Can this be a part of the `LogEventHandler` instead?